### PR TITLE
add container CI and remove validator_delegation requests

### PR DESCRIPTION
**Pull request by RicoToothless**

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -0,0 +1,32 @@
+FROM golang:1.16-alpine as builder
```
Comment (deni64k) [inline Dockerfile]: Please use go 1.18. Some previous versions have security vulns.

File: main.go [hunk 2]
```diff
@@ -165,10 +165,6 @@ func Execute(cmd *cobra.Command, args []string) {
 		ValidatorHandler(w, r, grpcConn)
 	})
 
-	http.HandleFunc("/metrics/validators", func(w http.ResponseWriter, r *http.Request) {
```
Comment (deni64k) [inline main.go]: I'm not sure why we remove it?

Comment (RicoToothless) [reply main.go]: Do we need to know all validators' status in the chain?

Comment (deni64k) [reply main.go]: If we don't call this endpoint, there's no need in removing it.

**Comment by deni64k** [comment]
LGTM beside two remarks

File: Dockerfile (hunk 1 above)
Comment (RicoToothless) [reply Dockerfile]: fixed

File: main.go (hunk 2 above)
Comment (RicoToothless) [reply main.go]: fixed

**Pull request merged**